server/test/controllers/FileControllerTest.java

```diff
@@ -13,18 +14,22 @@
 import models.StoredFileModel;
 import org.junit.Before;
 import org.junit.Test;
+import org.mockito.Mockito;
 import play.mvc.Http.Request;
 import play.mvc.Result;
+import services.settings.SettingsManifest;
 import support.ProgramBuilder;
 
 public class FileControllerTest extends WithMockedProfiles {
 
   private FileController controller;
+  private SettingsManifest mockSettingsManifest;
   private final Request request = fakeRequest().build();
 
   @Before
   public void setUp() {
     resetDatabase();
+    mockSettingsManifest = Mockito.mock(SettingsManifest.class);
```

Comment:
(praise) Thank you. We have too many tests relying on settings directly from the conf file.